### ISSUE TITLE
configure.pl will fail when cmake minor version gets >9

### ISSUE THREAD

**Issue by lasch** [IBM/CAST#412]
Trying to run configure.pl on a system with cmake 3.12.2 installed, it seems that it's caused by line 137:
```
    if(($ver cmp $minversion) > 0)
```
Since the default minversion string `3.2` is lexicographically bigger than the string `3.12.2`.

Reproduce:
 - run on a standard RHEL 7.5 with cmake3 installed and no other cmake under /opt/ibm or ~/coraltools/cmake/

Workaround:
 run configure.pl with additional argument `--mincmake=3.10`

**Comment by tgooding** [comment]
Looks like cmake has version comparators:

```
if(<variable|string> VERSION_LESS <variable|string>)
Component-wise integer version number comparison (version format is major[.minor[.patch[.tweak]]]).
if(<variable|string> VERSION_EQUAL <variable|string>)
Component-wise integer version number comparison (version format is major[.minor[.patch[.tweak]]]).
if(<variable|string> VERSION_GREATER <variable|string>)
Component-wise integer version number comparison (version format is major[.minor[.patch[.tweak]]]).
```

**Comment by lasch** [comment]
@tgooding isn't that capability of cmake too late when it already fails during configure.pl ?  Or is the build path via configure.pl something outdated and cmake should be used directly?

**Comment by tgooding** [comment]
I think this was resolved with commit 81e329e78d89a15202d88224339743578e1612e2.  Please verify.

**Comment by lasch** [comment]
works.
